templates/etc/icinga2/conf.d/commands.conf.j2

```diff
@@ -137,53 +137,125 @@ object NotificationCommand "mail-service-notification" {
   }
 }
 
-/*
- * If you prefer to use the notification scripts with environment
- * variables instead of command line parameters, you can use
- * the following commands. They have been updated from < 2.7
- * to support the new notification scripts and should help
- * with an upgrade.
- * Remove the comment blocks and comment the notification commands above.
- */
-
-/*
+{% if icinga2_master_twilio_sms_enabled|bool == true %}
+object NotificationCommand "twilio-sms-host-notification" {
+  command = [ ConfigDir + "/scripts/twilio-sms-host-notification.sh" ]
 
-object NotificationCommand "mail-host-notification" {
```

Comment:
Honestly, I would leave the default NotificationCommands mail-host-notification, mail-service-notification etc untouched and just add our own definitions. This keeps compatibility with people who want to use the standard features. Perhaps it would make sense to even split out our twilio notifications in a separate file like `commands-twilio.conf` which is only deployed when having twilio enabled and otherwise we use the standard commands.